RELEASE.md

```diff
@@ -0,0 +1,70 @@
+## Crafting a new release
+
+### Merge in upstream release to openshift repo:
+
+```bash
+# Check that a remote reference to openshift and upstream exists
+$ git remote -v | grep -e 'openshift\|upstream'
+openshift	git@github.com:openshift/knative-client.git (fetch)
+openshift	git@github.com:openshift/knative-client.git (push)
+upstream	https://github.com/knative/client.git (fetch)
+upstream	https://github.com/knative/client.git (push)
+
+# Create an openshift branch pointing to openshift client's master
+# (if not already done)
+$ git co -b openshift openshift/master
+
+# Go to scripts dir
+$ cd openshift/release
+
+# Create new release branch. Parameters are the upstream release tag
+# and the name of the branch to create
+$ ./create-release-branch.sh v0.9.0 release-v0.9.0
+
+# Push release back to openshift branch
+$ git push openshift release-v0.9.0
+```
+
+### Create a CI config and re-create Prow jobs
+
+* Create a fork of https://github.com/openshift/release (if not already done)
+* Create a new config YML for your release. Copy over `release-next.yaml` and adapt it by updating the image name from `knative-nightly` to a version specific name `knative-v0.9.0`
+
+```bash
+# Jump into the knative client config directory in the openshift/release
+$ cd ci-operator/config/openshift/knative-client
+
+# Copy over the nightly builds config to a release specific config
```

Comment:
add
#the name of the yaml ends with the release branch name to configure the CI against, here its `release-v0.9.0`